### DISCUSSION TITLE
Anexos e Apendices aparecem no indice mesmo sem estar presentes

### DISCUSSION THREAD

**Discussion by leoneldca** [joaomlourenco/novathesis#195] [Q&A] [answered]
Olá, 
Boa tarde!
Como devo proceder para que as páginas de anexos e apêndices parem de aparecer no índice, uma vez que estes arquivos não estão sendo utilizados? 
Dados
Template Uminho/EE versão [6.3.0]

**Comment by b-pereira** [comment]
Config/_files.tex e coloca os ficheiros de anexo e apêndice em comentário

**Reply by leoneldca** [reply]
Fiz isso mas não resolveu. Eles Continuaram a aparecer no índice. Não apareceram como link, mas apareceram como um texto em negrito.

**Reply by b-pereira** [reply]
Ou seja os títulos das partes **Apêndices** e **Anexos** aparecem no PDF, mesmo removendo quaquer referência. 

Vai a `NOVAthesisFiles/Schools/uminho/univ-defaults.ldf`. Aí tens a ordem do `mainmatter` em comentário. Remove os comentários e coloca o que tu queres assim:

```tex
\ntsetprintorder{mainmatter}{
chapters,        % Print document chapters
bib,             % Print the bibliography.  Change to
%                    %    “bib[title=⟨title⟩]” to redefine the title!
%   appendices,      % Print appendices, if any!
%   annexes,         % Print annexes, if any!
}

Por uma questão que completude, podes querer também tirar página de citação que se faz da mesma forma, mas no
`\ntsetprintorder{frontmatter}`

```

**Reply by joaomlourenco** [reply]
@leoneldca, tal como sugeriu o @b-pereira ,no ficheiro  “Config/6_files.tex” comenta as linhas onde se adiciona os apêndices e anexos, processa o ficheiro "template.tex" de novo (3 vezes e não apenas uma) e tudo deverá estar ok.  Se não estiver, diz qualquer coisa.

**Reply by leoneldca** [reply]
Infelizmente pra mim só deixa de aparecer se eu comentar appendix e annexes no arquivo novathesis.cls. 
Sei que não é o correto mas me vi sem alternativas. Uso a versão. 6.3 com TexStudio. Talvez no Overleaf não ocorra o problema

**Reply by leoneldca** [reply]
\ntsetprintorder{mainmatter}{
  chapters,        % Print document chapters
  bib,             % Print the bibliography.  Change to
                   %    “bib[title=⟨title⟩]” to redefine the title!
 %appendixes,      % Print appendixes, if any!
 %annexes,         % Print annexes, if any!
}

**Comment by joaomlourenco** [comment, marked as answer]
Atualiza o LaTeX (assumo que seja o miktex) e todos os seus pacotes!  Procura na net como se faz. Esse erro desaparecerá.
